### DISCUSSION TITLE
can't use comma or '-' on file name

### DISCUSSION THREAD

**Discussion by yogithesymbian** [exceljs/exceljs#2262] [Q&A]
my engine
```
"exceljs": "^4.3.0",
"express": "^4.17.1",
```

my code
```
res.setHeader('Content-Disposition', 'attachment; filename=' + `${fileName}`)
const generate = await workbook.xlsx.writeFile(
      `resources/static/assets/excel/${fileName}`
 )
```

work : `let fileName = 'XI 04 YOGI ARIF WIDODO 015 LOREM DAN IPSUM IPSUM LOREM LOREM IPSUM DESEMBER 2022.xlsx'`

not work : `let fileName = 'XI-04 YOG-ARIF-WIDODO-015 LOREM DAN IPSUM IPSUM LOREM LOREM IPSUM, DESEMBER 2022.xlsx'`

error : `net::ERR_RESPONSE_HEADERS_MULTIPLE_CONTENT_DISPOSITION`

anyone can help me, why can't naming with `,` or `-` on filename ?

my expected name / output is : **XI-04 YOG-ARIF-WIDODO-015 LOREM DAN IPSUM IPSUM LOREM LOREM IPSUM, DESEMBER 2022.xlsx**

**Comment by CHRISTINA1234m** [comment]
hi

**Comment by CHRISTINA1234m** [comment]
hi
